.travis.yml

```diff
@@ -40,11 +38,11 @@ install:
 
 script:
   - docker-compose exec web bash -c 'echo "$GOOGLE_APPLICATION_CREDENTIALS_JSON" > /tmp/credentials.json' # needs to be a path to a json file, so we shuffle...
-  - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true $TASK"
-  - if [ $TASK != "rake test:all" ]; then
-      echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml;
-      tail -n +2 -q ./test/reports/TEST*.xml >> output.xml;
-    fi
+  - docker-compose exec web bash -c "GENERATE_REPORT=true $TASK"
+# - if [ $TASK != "rake test:all" ]; then
+#     echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml;
+#     tail -n +2 -q ./test/reports/TEST*.xml >> output.xml;
+#   fi
```

Comment:
@jywarren @publiclab/reviewers What are these lines supposed to be doing? I'm not sure if they are related to coveralls or not

Reply:
They're related to Coveralls, they generate the XML to be pushed to Coveralls. You can remove those.

Reply:
Not sure about this. Dangerbot uses the xml too I think.

Reply:
These were intended to search through standard formatted JUnit test reports and copy them into the Dangerbot comment in the PR. They broke some time ago, but were a really nice way to help people see where errors were occurring. I'd like to leave them in so we can fix them again some day!